avivator/README.md

```diff
@@ -0,0 +1,52 @@
+# Avivator
+
+Avivator is a lightweight "batteries-included" WebGL viewer for remote imaging data, built around Viv.
+
+## System requirements
+
+Avivator has been tested with the following dependency versions:
+- Operating system: macOS 10.15.5
+- NodeJS 14.0.0
+- NPM 6.14.4
+- Firefox Developer Edition 84.0b8
```

Comment:
Do you also have chrome installed on your machine? It would be nice to have:

```
 - Firefox Developer Edition 84.0b8 / Chrome xx.x 
```

maybe even safari